From 073c2cf58636b0e830deeabbecea9b8ea2b1b279 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Mon, 1 Feb 2021 20:43:33 -0600
Subject: [PATCH 04/50] Change Turing Event template name.

This update changes the name of the IMA template from
ima-identity to ima-events, consistent with the renaming strategy
being pursued.
---
 security/integrity/ima/ima_template.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/integrity/ima/ima_template.c b/security/integrity/ima/ima_template.c
index 63c1f01ffc4e..3bd04801fcee 100644
--- a/security/integrity/ima/ima_template.c
+++ b/security/integrity/ima/ima_template.c
@@ -24,7 +24,7 @@ static struct ima_template_desc builtin_templates[] = {
 	{.name = "ima-sig", .fmt = "d-ng|n-ng|sig"},
 	{.name = "ima-buf", .fmt = "d-ng|n-ng|buf"},
 	{.name = "ima-modsig", .fmt = "d-ng|n-ng|sig|d-modsig|modsig"},
-	{.name = "ima-identity",.fmt = "actor|subject",
+	{.name = "ima-events",.fmt = "actor|subject",
 	 .genhash = ima_identity_genhash},
 	{.name = "", .fmt = ""},	/* placeholder for a custom format */
 };
-- 
2.31.1

